org.ekstep.navigation-1.0/renderer/plugin.js

```diff
@@ -18,9 +18,6 @@
     this.controllerPath = org.ekstep.pluginframework.pluginManager.resolvePluginResource(this._manifest.id, this._manifest.ver, "renderer/controller/navigation_ctrl.js");
     org.ekstep.service.controller.loadNgModules(this._templatePath, this.controllerPath);
 
-    EkstepRendererAPI.addEventListener("renderer:overlay:show", instance.showOrHideOverlay, instance);        
-    EkstepRendererAPI.addEventListener("renderer:content:start", instance.showOrHideOverlay, instance);
```

Comment:
Dont remove this code. In controller, it is not updating to plugininstance.customNavigationVisible  variable.